### PR TITLE
Service::copyAsChild() - $language param should be a string or null

### PR DESCRIPTION
**Pull request by blankse**
See https://github.com/pimcore/pimcore/issues/15428 and https://github.com/pimcore/admin-ui-classic-bundle/pull/143

### CODE REVIEW

File: models/Document/Service.php [hunk 1]
```diff
@@ -157,18 +157,9 @@ public function copyRecursive(Document $target, Document $source): Page|Document
...
-     * @param bool $enableInheritance
-     * @param bool $resetIndex
-     * @param bool $language
-     *
-     * @return Page|Document|PageSnippet
-     *
      * @throws ValidationException
      */
-    public function copyAsChild(Document $target, Document $source, bool $enableInheritance = false, bool $resetIndex = false, bool $language = false): Page|Document|PageSnippet
+    public function copyAsChild(Document $target, Document $source, bool $enableInheritance = false, bool $resetIndex = false, ?string $language = null): Page|Document|PageSnippet
```
Comment (dvesh3) [inline models/Document/Service.php:162]: maybe we should cast it [here](https://github.com/pimcore/admin-ui-classic-bundle/blob/4e547a79f37a2f0d531f09d6d386e8887fe30aa7/src/Controller/Admin/Document/DocumentController.php#L1027) and change the default value to null?

Comment (blankse) [reply models/Document/Service.php:162]: Did you see the PR https://github.com/pimcore/admin-ui-classic-bundle/pull/143 ?

Comment (dvesh3) [reply models/Document/Service.php:162]: Ohh, sorry i missed it 😄 should we cast it also?

Comment (blankse) [reply models/Document/Service.php:162]: Why should we cast it? the `$request->get()` method should return a string if it is a vaild language right?

Comment (dvesh3) [reply models/Document/Service.php:162]: I mean getting it specifically `$request->query->getString('language')`

Comment (blankse) [reply models/Document/Service.php:162]: @dvesh3 I changed it https://github.com/pimcore/admin-ui-classic-bundle/pull/143/commits/95e0fd1e40653e649633d1fbbac82333dcc9186a

**Review by dvesh3 — APPROVED** [review]

**Comment by dvesh3** [comment]
@blankse thanks!

**Pull request merged**